pkg/cable/wireguard/getconnections.go

```diff
@@ -141,9 +143,25 @@ func peerTrafficDelta(c *v1.Connection, key string, newVal int64) int64 {
 	return 0
 }
 
-// Save backendConfig[key]
-func savePeerTraffic(c *v1.Connection, lc, tx, rx int64) {
+// Save backendConfig[key] and export the metrics to prometheus
+func saveAndExportPeerTraffic(c *v1.Connection, lc, tx, rx int64) {
 	c.Endpoint.BackendConfig[lastChecked] = strconv.FormatInt(lc, 10)
 	c.Endpoint.BackendConfig[transmitBytes] = strconv.FormatInt(tx, 10)
 	c.Endpoint.BackendConfig[receiveBytes] = strconv.FormatInt(rx, 10)
+
+	endpointLabels := getLabelsFromEndpoint(&c.Endpoint)
+	timeCreated, _ := strconv.ParseInt(c.Endpoint.BackendConfig[timeCreated], 10, 64)
+	timeAlive := float64((time.Now().UnixNano() - timeCreated) / int64(time.Second))
```

Comment:
Prometheus suggest to use timestamps vs time since: https://prometheus.io/docs/practices/instrumentation/#timestamps-not-time-since

Reply:
the current shared layer use timestemps:
```go

func RecordConnection(cableDriverName string, localEndpoint, remoteEndpoint *submv1.EndpointSpec, status string, isNew bool) {
	labels := getLabels(cableDriverName, localEndpoint, remoteEndpoint)

	if isNew {
		connectionEstablishedTimestampGauge.With(labels).Set(float64(time.Now().Unix()))
	}
	
	labels[connectionsStatusLabel] = status
	connectionsGauge.With(labels).Set(1)
```